docs/userguide/transfer_learning.md

```diff
@@ -1,4 +1,41 @@
 # Transfer Learning
 
-This page will soon contain information about transfer learning.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers the possibility to mix data from multiple campaigns in order to accelerate
+optimization. 
+Using data from multiple campaigns is currently supported when using a Gaussian Process
+Surrogate model and is implemented by the [`TaskParameter`](baybe.parameters.categorical.TaskParameter).
+
+```{admonition} Terminology
+:class: note
+The term "Transfer Learning" is used in a lot of different ways.
+The act of combining the data of several contexts can be done via multiple models,
+shared architectures, special kernels in a single model and whatnot.
+We do not necessarily want to limit to any of these methods, even though BayBE currently
+offers only a single one.
+```
+
+## The role of ``TaskParameter``
```

Comment:
again this question: do we still need double backticks or not?

Reply:
I think they are not necessary in md files, I will change it here and will then check.

Reply:
They are in fact not necessary and were removed.